tutorials/nservicebus-101/nsb101-intro-paragraph.include.md

```diff
@@ -0,0 +1,3 @@
+The best way to get started with NServiceBus is to use it to build something realistic. In doing so you'll learn the architectural concepts behind the software, and start to learn its capabilities. In this tutorial, you'll be building a back end for a retail e-commerce system. You'll learn how to send asynchronous messages between processes, how to use the Publish/Subscribe pattern to decouple business processes, and the advantages of using reliable messaging to enable automatic retries after processing failures.
+
```

Comment:
@YifatS I'm conflicted on that. I want to keep it brief and get them started without a whole lot of why you should care (until we've shown something to make them care) since the marketing pages leading to this should have already been focused on that.